src/DynamoRevit/Models/RevitDynamoModel.cs

```diff
@@ -41,24 +42,65 @@ public class RevitDynamoModel : DynamoModel
         #region Events
 
         public event EventHandler RevitDocumentChanged;
+
         public virtual void OnRevitDocumentChanged()
         {
             if (RevitDocumentChanged != null)
                 RevitDocumentChanged(this, EventArgs.Empty);
         }
 
+        public event Action RevitDocumentLost;
+
+        private void OnRevitDocumentLost()
+        {
+            var handler = RevitDocumentLost;
+            if (handler != null) handler();
+        }
+
+        public event Action RevitContextUnavailable;
```

Comment:
I feel like some of these require some documentation.  What is the RevitContext?  When is it lost?